New topic for Listing CSPR (#242)
casper-network/docs: source/docs/casper/dapp-dev-guide/list-cspr.md

# Listing CSPR

> The Casper Network [whitepaper](../../../docs/casper/index.md) describes the capabilities of the network in detail. This document is tailored specifically for exchanges that wish to list the Casper token (CSPR) on their exchange. 

Reviewer: ```suggestion
The Casper Network [whitepaper](../../../docs/casper/index.md) describes the capabilities of the network in detail. This document is tailored specifically for exchanges that wish to list the Casper token (CSPR). 
```

I think the second use of exchange is redundant.

> CSPR is listed on over 25 exchanges worldwide. As a token, CSPR is very straightforward to integrate. The integration and testing can be completed by a developer in a few days. 

Reviewer: ```suggestion
CSPR is listed on over 25 exchanges worldwide. As a token, CSPR is very straightforward to integrate. Developers can complete integration and testing in a matter of days.
```
Suggestion for clarity.
